package.json

```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-edx.org@^1.3.2",
     "@edx/frontend-component-footer": "10.1.1",
-    "@edx/frontend-platform": "^1.8.0",
+    "@edx/frontend-platform": "1.8.1",
```

Comment:
Is there a reason to pin this at `1.8.1` instead of `^1.8.1`?

Reply:
LGTM pending an answer on this line.

Reply:
Let's keep that convo moving, but elsewhere. I don't think pinning/not pinning is a block to removing the vulnerable version.